MediaPlayer/MediaControls.js

```diff
@@ -206,70 +214,69 @@ const MediaControls = kind({
 	},
 
 	computed: {
-		badge: ({repeat}) => badges[repeat]
+		badge: ({repeat}) => badges[repeat],
+		className: ({type, styler}) => styler.append(type)
 	},
 
-	render: ({badge, disabled, menuIcon, nextTrackIcon, onNext, onPause, onPlay, onPrevious, onRepeat, onShuffle, paused, pauseIcon, playIcon, previousTrackIcon, repeatIcon, shuffle, shuffleIcon, ...rest}) => {
+	render: ({badge, disabled, menuIcon, nextTrackIcon, onNext, onPause, onPlay, onPrevious, onRepeat, onShuffle, paused, pauseIcon, playIcon, previousTrackIcon, repeatIcon, shuffle, shuffleIcon, type, ...rest}) => {
 		return (
 			<Container {...rest}>
-				<Button
-					aria-label={$L('Repeat')}
-					backgroundOpacity="transparent"
-					badge={badge}
-					css={css}
-					disabled={disabled}
-					icon={repeatIcon}
-					onClick={onRepeat}
-					size="large"
-				/>
-				<Button
-					aria-label={$L('Shuffle')}
-					backgroundOpacity="transparent"
-					className={shuffle ? css.repeat : ''}
-					css={css}
-					disabled={disabled}
-					icon={shuffleIcon}
-					onClick={onShuffle}
-					size="large"
-				/>
+				{type === 'full' ?
```

Comment:
Why it has a different pattern from the others?

Reply:
I changed to this pattern based on the previous review comment

Reply:
oh, right!